apps/src/dropletUtils.js

```diff
@@ -166,12 +166,11 @@ standardConfig.blocks = [
   {func: 'notOperator', block: '!__', category: 'Math'},
   // randomNumber_max has been deprecated
   // {func: 'randomNumber_max', block: 'randomNumber(__)', category: 'Math' },
-  // Note: We use randomNumber as our base docFunc here so that we get the benefits of param descriptions
   {
     func: 'randomNumber_min_max',
     block: 'randomNumber(1, 10)',
     category: 'Math',
-    docFunc: 'randomNumber'
+    docFunc: 'randomNumber_min_max'
```

Comment:
We have as separate documentation page for randomNumber_min_max which covers the params. We should use it here.